res/css/views/elements/_AccessibleButton.scss

```diff
@@ -16,152 +16,152 @@ limitations under the License.
 
 .mx_AccessibleButton {
     cursor: pointer;
+}
 
-    .mx_AccessibleButton_disabled {
```

Comment:
Why not use the `&` selector here?

e.g `&.mx_AccessibleButton_disabled {`

Reply:
I thought just reverting to the [latest one](https://github.com/luixxiul/matrix-react-sdk/blob/d5eb4d45dad7b7427607e1dcd10572bfab87603d/res/css/views/elements/_AccessibleButton.scss) before applying SCSS style nesting should work.

> Why not use the `&` selector here?

I think line 21 below should take care of that.

Reply:
Yes, just wondering why you're removing the nesting when it can made work

Reply:
I guess you're right. I'm pushing another commit.